doc/releases/changelog-dev.md

```diff
@@ -26,6 +26,10 @@
 
 <h3>Improvements</h3>
 
+* All drawing methods changed their default value for the keyword argument `show_matrices`
+  to `True`. This allows quick insights into broadcasted tapes for example.
+  [(#3920)](https://github.com/PennyLaneAI/pennylane/pull/3920)
+
```

Comment:
While a very minor breaking change that isn't going to break anyone's code, this is still a breaking change and should go under that section.

Reply:
Sorry, I had added it a second time to breaking changes as well but forgot to push :/